pandas/tests/series/indexing/test_where.py

```diff
@@ -309,7 +307,17 @@ def test_broadcast(size, mask, item, box):
     )
 
     s = Series(data)
-    s[selection] = box(item)
+
+    if selection.sum() != 1:
```

Comment:
hmm maybe make this another test

it's really confusing u r boxing first then setting it after

Reply:
sure, updated